cmd/kfulint/testdata/unexported-call/checker_tests.go

```diff
@@ -0,0 +1,17 @@
+package checker_test
+
+type foo struct{}
+
+func (f foo) unexported() int { return 1 }
+func (f foo) Exported() int   { return 1 }
+
+func getFoo() foo { return foo{} }
+
+func f() {
+	var fo foo
+	///: getFoo().unexported() should be exported
+	getFoo().unexported()
+	///: fo.unexported() should be exported
+	fo.unexported()
+	fo.Exported()
+}
```

Comment:
Need a test case for `foo` method that calls unexported methods and gets no warnings.